src/test/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentServiceSpec.scala

```diff
@@ -88,4 +88,34 @@ class ConsignmentServiceSpec extends AnyFlatSpec with MockitoSugar with Matchers
 
     response should be(None)
   }
+
+  "consignmentHasFiles" should "return true when files already associated with provided consignment id" in {
+    val fixedUuidSource = new FixedUUIDSource()
```

Comment:
Do we need to use FixedUUIDSource and UUID.randomUUID in the same test?

To be honest, I'd recommend using fixed UUIDs in unit tests. That way we don't have to worry about the state of FixedUUIDSource, or random test failures from a random data source*. It also means that if you see a UUID in the output of a failed test, you can track it back to a specific bit of test data.

\* Random test data is really useful in some circumstances, but you have to be able to re-run your tests with the same random seed, which we haven't set up here.

Reply:
The ConsignmentService constructor requires a UUID source which the FixedUUIDSource gives